### ISSUE TITLE
Invariance of phase - it applies to any periodic phenomena, not just to plane monochromatic waves. Right?

### ISSUE THREAD

**Issue by johanley** [johanley/special-relativity-core#79]
Picture an electric clock.
You plug it in, it runs for a long while, then you unplug it.
The phase corresponds, say to the reading of the minute hand. The phase is not constrained to be in the range [0..2pi).

The total change in the phase, between the plug-in event and the plug-out event, is an objective fact. It doesn't depend on the coordinate frame in any way.

Take a single event, when the minute hand is at 3. That's an objective fact. Every grid must agree on that fact for that event. Grids differ in their coords for the event, but that's another matter. No matter what the grid's coords for the event, the phase MUST say that "the minute hand is at 3 for this event in space-time."

The phase of a periodic phenomenon is invariant for any periodic phenomenon, not just for plane monochromatic waves.

In short, the phase describes a condition in the world, so by definition everyone must agree on that condition.

**Comment by johanley** [comment]
I added a remark on this in the Readme.
